src/connectors/connector_transcoder.workflows.json

```diff
@@ -0,0 +1,32 @@
+{
+  "main": {
+    "steps": [
+      {
+        "init": {
+          "assign": [
+            {
+              "project": "${sys.get_env(\"GOOGLE_CLOUD_PROJECT_ID\")}"
+            },
+            {
+              "location": "${sys.get_env(\"GOOGLE_CLOUD_LOCATION\")}"
+            }
+          ]
+        }
+      },
+      {
+        "get_list": {
+          "call": "googleapis.transcoder.v1.projects.locations.jobs.list",
+          "args": {
+            "parent": "${\"projects/\" + project + \"/locations/\" + location}"
+          },
+          "result": "response"
+        }
+      },
+      {
+        "the_end": {
+          "return": "${response}"
+        }
+      }
+    ]
+  }
+}
```

Comment:
just add a new line at the end of this yaml

Reply:
This Json is auto-generated by the tojson.sh script. I think we can leave it unchanged to match existing patterns. Thanks.